### PR TITLE
MockInteractions has moved to a better place

### PR DESCRIPTION
**Pull request by notwaldorf**
From core-focusable to iron-test-helpers. I've also:
- cleaned up the `paper-ripple` and `layout` deps in `bower.json`
- caught up with the updates in `iron-doc-viewer`

### CODE REVIEW

**Comment by notwaldorf** [comment]
@cdata PTAL

File: index.html [hunk 1]
```diff
@@ -20,16 +20,10 @@
...
-  <style>
-    body {
-      margin: 16px;
-    }
-  </style>
-
 </head>
 <body>
 
-  <iron-doc-viewer src="paper-radio-button.html"></iron-doc-viewer>
```
Comment (cdata) [inline index.html:32]: Where did the `src` go?

**Comment by cdata** [comment]
One bug to resolve, otherwise LGTM :+1:

File: index.html (hunk 1 above)
Comment (notwaldorf) [inline index.html:32]: `<src>` is sooooo yesterday. `<iron-doc-viewer>` picks it up automagically now. :)

(see https://github.com/PolymerElements/iron-doc-viewer/blob/master/iron-doc-viewer.html#L39)

**Pull request merged**